src/digitKeyboard.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+import { EventDispatcher } from './eventDispatcher.js';
+
+/**
+ *  DigitKeyboard sets the digits on the screen.
+ */
+export class DigitKeyboard extends EventDispatcher {
+
+    /**
+     *  Constructor.  
+     */
+    constructor() {
+        super();
+        this.bindEvents();
```

Comment:
Similar like for fields, we use underscore for private functions. Those are the functions which are going to be invoked only from another function in this class, not from the outside. For example `_bindEvents()` will never be called from `Calculator`, they will be only called from constructor. Please rename.